hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -1060,6 +1062,20 @@ solCom = \case
       [ solSet ("uint256" <+> actBalV) bal
       , solSet (solMemVar dv) ite
       ]
+  DL_Let _ (DLV_Let _ dv) (DLE_ContractFromAddress _at addr) -> do
+    maybeMap <- case maybeT T_Address of
```

Comment:
This should be `T_Contract`, that's the source of your struct mismatch error

Reply:
Oof, yes, that's pretty obvious in hindsight, thanks.